lib/livebook/notebook/cell.ex

```diff
@@ -18,6 +18,8 @@ defmodule Livebook.Notebook.Cell do
 
   @type indexed_output :: {non_neg_integer(), Livebook.Runtime.output()}
 
+  @type comment :: %{user: Livebook.Users.User.t(), message: String.t()}
```

Comment:
We don't need to store user id, because it's temporary. I'm not even sure if we should store the color, because then they change it it may be confusing that it's not the same. Usage-wise having just name, message and a timestamp should do the job. @josevalim thoughts?

Reply:
So we would not display user avatars I guess?